packages/components/stories/autocomplete/README.md

```diff
@@ -0,0 +1,72 @@
+# AutoComplete Component
+
+Advanced `AutoComplete` component.
+
+
```

Comment:
Can we add **Supported events** here?
`AutoComplete` supports key press events as well as onClick events.